src/rust/iced-x86/src/code.rs

```diff
@@ -41705,4 +41705,43 @@ impl Code {
 			_ => false,
 		}
 	}
+
+	#[must_use]
+	#[inline]
+	pub(crate) fn is_string_op(self) -> bool {
```

Comment:
This should be public (and it needs docs). You need to add this attribute too `#[cfg(feature = "instr_info")]`